app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Service catalog
+subtitle: Track every service across your architecture
+content-type: explanation
+---
+
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization, both {{site.base_gateway}} services and external integration resources.
```

Comment:
I worry that mentioning Gateway Services this early in the page might confuse readers into (incorrectly) thinking that this product is a catalog of Gateway Services.

Typically, I've gone with something like "Kong Konnect’s Service Catalog is designed to offer a comprehensive list of all services running in your organization. This catalog represents the single source of truth of your organization’s service inventory and their dependencies. It includes services that are powered by Kong Gateway and those that are not."